src/lib/services/feature-toggle-service.ts

```diff
@@ -369,7 +369,7 @@ class FeatureToggleService {
         featureStrategy: IFeatureStrategy,
         segments: ISegment[] = [],
     ): Saved<IStrategyConfig> {
-        return {
+        let result: Saved<IStrategyConfig> = {
```

Comment:
const since you're not re-assigning this variable only modifying one of the properties